### ISSUE TITLE
REST API

### ISSUE THREAD

**Issue by sevzero** [sevzero/honeybadger#10]
A framework needs to be put in place to allow API calls to be made

**Comment by morallo** [comment]
This would be a great feature for integration. Are there any plans for it?

**Comment by sevzero** [comment]
Definitely. It shouldn't be too much work so I'll take a look at this soon.
